.github/workflows/push-job-sched-jar.yml

```diff
@@ -0,0 +1,47 @@
+name: Upload Job-Scheduler Jar to Maven
```

Comment:
@zengyan-amazon we are publishing SPI not job scheduler jar right?
If so @rishabh6788 can you update name since it can be confusing

Reply:
> @zengyan-amazon we are publishing SPI not job scheduler jar right?
> If so @rishabh6788 can you update name since it can be confusing

Yes, we publish SPI to Maven central, as it is the dependency of other plugins like ISM.
Job scheduler plugin is a installable zip which won't be consumed, we don't publish it

Reply:
Updated the name